share/completions/ps2pdf14.fish

```diff
@@ -0,0 +1 @@
+complete -c ps2pdf14 --wraps=ps2pdf
```

Comment:
What are these? Old versions of this command?

Reply:
These ps2pdf scripts are installed by the `ghostscript` package. Currently `ps2pdf` targets PDF version 1.4, however it may change in a new version. The other scripts target specific versions of PDF such as 1.2, 1.3, etc.